app/schema.py

```diff
@@ -165,12 +182,97 @@ def mutate(root, info, other_user):
+            friends_contacts_list = [ str(fri_obj.friend.contact) for fri_obj in Friend.objects.filter(current_user=info.context.user) ]
+            for contact in contacts_list:
+                if contact not in friends_contacts_list:
+                    return CreateGroup(ok=False, message="Some contacts are not of your friends'", group=None)
+            # create group with the members(selected friends)
+            group = Group.objects.create(group_name=group_name, group_admin=info.context.user)
+            for contact in contacts_list:
+                member, created = Member.objects.get_or_create(user=User.objects.get(contact=contact))
+                group.group_member.add(member.id)
+            return CreateGroup(ok=True, message="Group created successfully", group=group)
+
+
+class AddMembers(graphene.Mutation):
+    class Arguments:
+        group_name = graphene.String()
+        contacts_list = graphene.List(graphene.String)
+    message = graphene.String()
+    ok = graphene.Boolean()
+    group = graphene.Field(GroupType)
+
+    @staticmethod
+    @login_required
+    def mutate(root, info, group_name, contacts_list):
+        try:
+            # to remove duplicate contacts
+            contacts_list = list(set(contacts_list))
+            friends_contacts_list = [ str(fri_obj.friend.contact) for fri_obj in Friend.objects.filter(current_user=info.context.user) ]
+            for contact in contacts_list:
+                if contact not in friends_contacts_list:
+                    return AddMembers(ok=False, message="Some contacts are not of your friends'", group=None)
+            group = Group.objects.get(group_name=group_name, group_admin=info.context.user)
+            members_list = [ member_obj.user.contact for member_obj in group.group_member.all() ]
+            for contact in contacts_list:
+                if contact not in members_list:
+                    member, created = Member.objects.get_or_create(user=User.objects.get(contact=contact))
+                    group.group_member.add(member.id)
+            return AddMembers(ok=True, message="Successfully added new members", group=group)
+        except Group.DoesNotExist:
+            return AddMembers(ok=False, message="Group doesn't exist", group=None)
+
+
+class RemoveMembers(graphene.Mutation):
+    class Arguments:
+        group_name = graphene.String()
+        contacts_list = graphene.List(graphene.String)
+    message = graphene.String()
+    ok = graphene.Boolean()
+    group = graphene.Field(GroupType)
+
+    @staticmethod
+    @login_required
+    def mutate(root, info, group_name, contacts_list):
+        try:
+            # to remove duplicate contacts
+            contacts_list = list(set(contacts_list))
+            group = Group.objects.get(group_name=group_name, group_admin=info.context.user)
+            members_list = [ member_obj.user.contact for member_obj in group.group_member.all() ]
+            for contact in contacts_list:
```

Comment:
Remove in bulk, and you can also pass the contacts_list directly to the remove method.